src/msw/notebook.cpp

```diff
@@ -499,6 +499,25 @@ void wxNotebook::SetTabSize(const wxSize& sz)
     ::SendMessage(GetHwnd(), TCM_SETITEMSIZE, 0, MAKELPARAM(sz.x, sz.y));
 }
 
+wxRect wxNotebook::GetTabRect(size_t n) const
+{
+    wxRect r;
+
+    if (GetPageCount() > 0)
+    {
+        RECT rect;
+        if (TabCtrl_GetItemRect(GetHwnd(), n, &rect))
+        {
+            r.x = rect.left;
```

Comment:
I would use `r = wxRectFromRECT(rect);`  instead of all these four lines.

Reply:
Fixed. Didn't know about this funtion. :)